File: typescript/src/price-dao.ts
export class PriceDao {
    constructor(private connection: any) {

    }

    public async findBasePrice(liftPassType) {
        const result = (await this.connection.query(
            'SELECT cost FROM `base_price` ' +
            'WHERE `type` = ? ',
            [liftPassType]))[0][0]
        return result
    }

    public async findAllHolidays() {
        const holidays = (await this.connection.query(
            'SELECT * FROM `holidays`'
        ))[0]
        return holidays
    }
}

File: python/src/services/price_dao.py

from typing import TYPE_CHECKING

from python.src.services.cost import Cost

if TYPE_CHECKING:
    from sqlite3 import Cursor


class PriceDao:
    def __init__(self, connection):
        self.connection = connection
        self.cursor = self.connection.cursor()

    def find_holidays(self) -> "Cursor":
        return self.cursor.execute('SELECT * FROM holidays')

    def find_base_price(self, lift_pass_type: str) -> Cost:
        self.cursor.execute(f'SELECT cost FROM base_price '
                            + 'WHERE type = ? ', (lift_pass_type,))
        row = self.cursor.fetchone()
        result = {"cost": row[0]}
        return Cost(result["cost"])
